build/evergreen_config_lib/tasks.py

```diff
@@ -251,7 +251,7 @@ def __init__(self, task_name, extra_commands, orchestration=True, **kwargs):
                           exit 123
                         fi''')]),
     CompileTask('compile-tracing',
-                TRACING='ON'),
+                TRACING='ON', CFLAGS='-Werror -Wno-cast-align'),
```

Comment:
Copying the `-Wno-cast-align` from the debug-compile-with-warnings task. I believe we expect those when compiling with the alignment attributes.